src/cace/ari/text_str.yac

```diff
@@ -262,7 +263,18 @@ ssp-literal:
             }
             else
             {
-                //FIXME where to convert (and check) type?
+              ari_t tmp;
+              ari_init(&tmp);
+              int err = amm_type_convert(amm_type_get_builtin(ARI_TYPE_TP), &tmp, found_r);
+              if (err) 
+              {
+                ari_deinit(found_r);
```

Comment:
Please add `ari_deinit(&tmp)` for consistency

Reply:
No problem, added this call.